subsystems/LinearServo.java

```diff
@@ -0,0 +1,45 @@
+package frc.team670.mustanglib.subsystems;
+
+import edu.wpi.first.wpilibj.Servo;
+
+
+/**
+ * Represents a Linear Servo
+ * 
+ * @author AkshatAdsule
+ */
+public class LinearServo {
+    
+    private final Servo actuator;
+
+    /**
+     * Creates a new LinearServo
+     * @param channel the PWM port 
+     */
+    public LinearServo(int channel) {
+        actuator = new Servo(channel);
+        actuator.setBounds(2.0, 1.8, 1.5, 1.2, 1.0);
```

Comment:
These values are specific to the AndyMark linear actuator that we got, right? Maybe change the class name to indicate that. Or better maybe have an enum that specifies the type of linear actuator (currently only the AndyMark L16-R https://www.andymark.com/products/actuator-l16-r-50mm-stroke-150-1-6v assuming that's the one we got) and then set these values based on that to hopefully avoid screwups and provide a way to extend this in the future if we try other linear actuators

Reply:
added in [3e19ba0](https://github.com/HHS-Team670/MustangLib/pull/10/commits/3e19ba056c75e327483ceac2aaf8085f83811e55) and [3a9447f](https://github.com/HHS-Team670/MustangLib/pull/10/commits/3a9447f073054c7865c3a904e26c3d78b5305e21)